src/app/shared/autocomplete/autocomplete.component.less

```diff
@@ -0,0 +1,53 @@
+@import "../../variables.less";
+
+.autocomplete-section {
+  width: 40vw;
+  margin: 5% auto;
+  border: 2px solid @primary-red-color;
+  border-radius: 20px;
+  padding: 2%;
+
+  .form-group {
+    position: relative;
+    margin: 3%;
+  }
+
+  .search-button {
+    margin: 5% 0;
+    width: 100%;
+  }
+}
+
+.city-dropdown-container {
+  position: absolute;
+  z-index: 1;
+  width: 100%;
+  max-height: 150px;
+  background: @white_deep;
+  border: 0.5px solid darkgray;
+  overflow-y: auto;
+
+  .city-text {
+    padding: 2% 5%;
+    color: @primary-red-color;
+    font-weight: 500;
+
+    &:hover {
+      background: lightgrey;
+    }
+  }
+}
+
+.autocomplete-row {
+  display: flex;
+
+  & .autocomplete-col {
+    display: block;
+
+    & #pickUpTime,
```

Comment:
It would be better to use classes instead of ID's for styling

Reply:
Changed to classes